test/e2e/signals_test.go

```diff
@@ -141,7 +141,7 @@ func (s *SignalsSuite) TestSignaled() {
 		Workflow("@testdata/signaled-workflow.yaml").
 		When().
 		SubmitWorkflow().
-		WaitForWorkflow().
+		WaitForWorkflow(fixtures.ToBeCompleted).
```

Comment:
the default is [`ToBeDone`](https://github.com/argoproj/argo-workflows/blob/6b16f24c97e4b9c792e18835304e517f2d0ddd0f/test/e2e/fixtures/when.go#L257), which [is a subset of `ToBeCompleted`](https://github.com/argoproj/argo-workflows/blob/6b16f24c97e4b9c792e18835304e517f2d0ddd0f/test/e2e/fixtures/when.go#L218) which should be equivalent in this case, so I wouldn't think this would change anything